Client/Frontend/Toolbar+URLBar/TabLocationView.swift

```diff
@@ -284,7 +284,11 @@ class TabLocationView: UIView, FeatureFlaggable {
             return
         }
         let product = ShoppingProduct(url: url)
-        shoppingCartButton.isHidden = !product.isShoppingCartButtonVisible || tab.isPrivate
+        let shouldHideButton = !product.isShoppingCartButtonVisible || tab.isPrivate
+        shoppingCartButton.isHidden = shouldHideButton
+        if !shouldHideButton {
+            TelemetryWrapper.recordEvent(category: .action, method: .view, object: .shoppingCartButton)
```

Comment:
I believe the icon will be changing so we might not want to use 'shopping cart'

Reply:
We can change it when we update the icon. I'll add that too the task we have for this.